slack_bolt/middleware/authorization/async_multi_teams_authorization.py

```diff
@@ -96,7 +96,7 @@ async def async_process(
                     "Although the app should be installed into this workspace, "
                     "the AuthorizeResult (returned value from authorize) for it was not found."
                 )
-                if req.context.response_url is not None:
+                if req.context.response_url is not None and callable(req.context.respond):
```

Comment:
I don't think this should change the current behavior of the code base and it should make it more resilient, but asserting callable is another option
 ```suggestion
 if req.context.response_url is not None:
         assert callable(req.context.respond), "req.context.respond must be callable"
```

Reply:
Sounds good 💯 I will remove the `assert` changes and add `type: ignore` statements when needed